main.py

```diff
@@ -13,45 +13,67 @@ class JournalEntry:
         title: str
         content: str
         date: datetime
-
-dict_journal_entry = {}
+        
+def add_entry(title: str, content: str) -> JournalEntry:
+    new_entry = JournalEntry(title=title,content=content,date=datetime.now())
+    #new_entry_dict = asdict(new_entry)
+    #new_entry = new_entry_dict
+    #print(new_entry)
+    return new_entry
+    #print(entries)
+    
+    #entries = load_entries()
+    #entries.append(new_entry_dict)
 
         
-def load_entries() -> str: 
-        with open('journal.json', "r") as user_file:
-         journal_content = user_file.read()
-        print(journal_content)
-"""Load journal entries from the JSON file."""
+        #Create a new journal entry.
 
-    
-def save_entries(entries: dict[JournalEntry]) -> str:
-        with open("journal.json","w") as json_file:
-             json.dump(dict_journal_entry,json_file)
-        """Save journal entries to the JSON file."""
+def save_entries(title: str, content: str) -> None: #entries: list[JournalEntry] #use asdict() in save_entries function
+        new_entry = add_entry(title,content)
```

Comment:
this function is only to save the entries, well done below, remove `add_entry`, this should not go here